web/modules/custom/do_base/do_base.services.yml

```
@@ -0,0 +1,6 @@
+services:
+  do_base.twig.icon_assets:
+    class: Drupal\do_base\Twig\IconAssetsExtension
+    arguments: ['@theme.manager']
+    tags:
+      - { name: twig.extension }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)